docs/source/gbd2020_models/risk_exposures/wasting_state_exposure/index.rst

```diff
@@ -490,7 +490,66 @@ pool.
 Here we include equations for the transition probabilities, and in the section 
 that follows we will detail how to calculate all the variables used
 
-.. list-table:: Wasting transition probability equations
+.. list-table:: NEW WASTING TRANSITION PROBABILITY EQNS
+   :widths: 5 15 10 10
+   :header-rows: 1
+
+   * - Variable
+     - Equation
+     - Description
+     - Source
+   * - r2
+     - -ap0*f2/ap1 - ap0*f3/ap1 - ap0*f4/ap1 - t1 + ap2*d2/ap1 + ap2*i1/ap1 + ap3*d3/ap1 + ap4*d4/ap1
+     - Probability of remission into cat 2 from cat 1
+     - System of equations
+   * - i1
+     - ap0*f2/ap2 + ap0*f3/ap2 + ap0*f4/ap2 + ap1*r2/ap2 + ap1*t1/ap2 - d2 - ap3*d3/ap2 - ap4*d4/ap2
+     - Probability of incidence into cat 1 from cat 2
+     - System of equations
+   * - i2
+     - ap0*f3/ap3 + ap0*f4/ap3 + ap1*t1/ap3 + ap2*r3/ap3 - d3 - ap4*d4/ap3
+     - Probability of incidence into cat 2 from cat 1
+     - System of equations
+   * - i3
+     - ap0*f4/ap4 + ap3*r4/ap4 - d4
+     - Probability of incidence into cat 3 from cat 4
+     - System of equations
+   * - r2
+     - 0.512/60.5
+     - Probability of remission into cat 2 from cat 1 (untreated)
+     - Nicole's calculations; also referred to as r2ux (get lit source!)
+   * - r3
+     - 0.512/63 + 0.488/41.3
```

Comment:
Can we get these as variables, rather than numbers?

Reply:
those numbers come directly from nicole's papers. i can give it to you as
r3 = treatment_coverage * time_to_treated_recovery + (1-treatment_coverge)*time_to_untreated_recovery

is that preferable / what you mean @rmudambi ?

Reply:
ah, these equations are in the wasting treatment PR! (which I will merge today)